packages/edit-site/src/components/editor/style.scss

```diff
@@ -17,3 +17,60 @@
 	display: flex;
 	justify-content: center;
 }
+
+.edit-site-editor__view-mode-toggle {
+	/* stylelint-disable -- Disable reason: View Transitions not supported properly by stylelint. */
+	view-transition-name: toggle;
+	/* stylelint-enable */
+	position: fixed;
```

Comment:
Something about this commit caused the icon to sit low in distraction free. Commenting here because removing the fixed position resolves it in my testing

https://github.com/user-attachments/assets/8f96b692-eb01-4543-99ea-9eebacc2f1b2

Reply:
Good catch

Reply:
Fix in #64261